packages/cli/package.json

```diff
@@ -28,6 +28,7 @@
     "algosdk": "^1.19.0",
     "cli-ux": "^6.0.9",
     "ejs": "^3.1.6",
+    "ethers": "^5.7.0",
```

Comment:
Does `@subql/types-ethereum` or `@subql/common-ethereum` have this as a dependency? If so maybe we could make this a peer dependency